resilience4j-framework-common/src/main/java/io/github/resilience4j/common/bulkhead/configuration/CommonBulkheadConfigurationProperties.java

```diff
@@ -148,7 +148,7 @@ public Duration getMaxWaitDuration() {
 
         public InstanceProperties setMaxWaitDuration(Duration maxWaitDuration) {
             Objects.requireNonNull(maxWaitDuration);
-            if (maxWaitDuration.toMillis() < 0) {
+            if (maxWaitDuration.isNegative()) {
```

Comment:
Before change it would not reject `Duration.ofNanos(-999999)` which returns 0 for `toMillis()`